test_board.py

```diff
@@ -9,12 +9,21 @@ def test_board_state_is_a_list():
 def test_when_a_user_begins_a_game_a_3X3_board_is_created():
     game_board = Board()
     game_board.new_game()
-    assert game_board.board_state == [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+    assert game_board.board_state == [
+                                        "", "", "", 
+                                        "", "", "", 
+                                        "", "", ""
+                                     ]
 
 def test_when_a_user_supplies_4_rows_a_4x4_board_is_created():
     game_board = Board()
     game_board.new_game(4)
-    assert game_board.board_state == [" ", " ", " ", " ", " ", " ", " ", " ", " ", " ", " ", " ", " ", " ", " ", " "]
+    assert game_board.board_state == [
+                                        "", "", "", "", 
+                                        "", "", "", "", 
+                                        "", "", "", "", 
+                                        "", "", "", ""
+                                     ]
 
 # must test 'sad' path for new_game by passing in a value less then 3
```

Comment:
Why wouldn't 2 work? or 1 or 0 for that matter?

Reply:
also let's try to not leave comments.

Reply:
I'll remove my comments.
There's no reason at this point why any number wouldn't work. I was planning on adding a validation to prevent the user from selecting anything less than 3.

Reply:
So let's say I wanted to test a board with size 100 how would you do that?